src/core/syscalls/business_logic_syscall_handler.rs

```diff
@@ -30,11 +30,11 @@ use num_traits::{One, ToPrimitive, Zero};
 use std::borrow::{Borrow, BorrowMut};
 
 //* -----------------------------------
-//* BusinessLogicHandler implementation
+//* DeprecatedBLSyscallHandler implementation
 //* -----------------------------------
-
+// This is the old version of the BusinessLogicSyscallHandler
```

Comment:
Maybe this could be a docstring instead of a normal comment to more formally document the struct

Reply:
Makes sense. Thanks.
```suggestion
/// This is the old version of the BusinessLogicSyscallHandler
```